fix(ui): fix message input buttons not customizable (#2119)
GetStream/stream-chat-flutter: packages/stream_chat_flutter/CHANGELOG.md

## Upcoming

🐞 Fixed

> - [[#2118]](https://github.com/GetStream/stream-chat-flutter/issues/2118) Fixed message input
>   buttons not being able to customized.

Reviewer: ```suggestion
- [[#2118]](https://github.com/GetStream/stream-chat-flutter/issues/2118) Fixed an issue where message input buttons could not be customized.
```
